src/js/components/award/shared/activity/ContractGrantActivity.jsx

```diff
@@ -46,17 +62,92 @@ const ContractGrantActivity = ({
             window.removeEventListener('resize', handleWindowResize);
         };
     }, []);
+    /**
+     * X Translation
+     * We are positioning the potential award amount line tooltip centered.
+     * Therefore, we must halve the width to get the center of the graph and add 8
+     * (since 8 is half the pointer for the tooltip that is auto adjusted in the tooltip
+     * wrapper) and subtract half of the tooltip (since the tooltip position draws left to right)
+     * Y Translation
+     * We position with respect to the y position and add 8 for half the point and padding bottom.
+     */
+    const potentialAwardAmountLineTooltipData = (data) => ({
+        tooltipPosition: 'bottom',
+        styles: { // 8px is half the tooltip pointer, data.position is the y-position
+            transform: `translate(${((data.x2 / 2) + 8) - (defaultTooltipWidth / 2)}px,${data.position + 8 + defaultPadding.bottom}px)`,
+            position: 'absolute'
+        },
+        tooltipComponent: <RectanglePercentVizTooltip
+            title="Current Potential Award Amount"
+            amount={formatMoney(totalObligation)} />
+    });
+
+    const verticalLinesTooltipData = (data, text) => {
+        const date = moment(data.value);
+        const dateFormatted = date.format("MM/DD/YYYY");
+        const today = moment(Date.now());
+        const dateText = today.isAfter(date) ?
+            `${dateFormatted} (${convertDatesToRange(date, today)} ago)` :
+            `${dateFormatted} (${convertDatesToRange(today, date)} from today)`;
+        setThisLineOrTextIsHovered(text);
+        return {
+            styles: { // y position is in the middle of the line
+                position: 'absolute',
+                transform: `translate(${((data.position))}px,${(height / 2) - defaultPadding.bottom}px)`
+            },
+            tooltipComponent: <RectanglePercentVizTooltip
+                title={`${text} Date`}
+                amount={dateText}
+                tooltipPosition="right" />
+        };
+    };
+
+    const handleTooltipData = (data, text) => {
+        let tooltipInfo = null;
+        if (!text) { // potential award amount line
+            tooltipInfo = potentialAwardAmountLineTooltipData(data);
+        }
+        else { // all other award lines
+            tooltipInfo = verticalLinesTooltipData(data, text);
+        }
+        setTooltipData(tooltipInfo);
+        setShowTooltip(true);
+    };
+
+    const showHideTooltip = (data, text) => {
+        // hide tooltip
+        if (!data && showTooltip) {
+            setShowTooltip(false);
+            setThisLineOrTextIsHovered('');
+        }
+        else {
+            handleTooltipData(data, text);
+        }
+    };
 
     return (
-        <div ref={divReference} className="contract-grant-activity-visualization">
+        <div ref={divReference} className="award-amounts-viz contract-grant-activity-visualization">
+            {showTooltip && <TooltipWrapper
```

Comment:
You could give the TooltipWrapper children for the hover area that should trigger it rather than doing this.